packaging/leapp.spec

```diff
@@ -94,14 +96,51 @@ BuildRequires:  python2-setuptools
 Provides: leapp-framework = %{framework_version}
 Requires: leapp-framework-dependencies = %{framework_dependencies}
 
+# We expect people to manually remove this rpm if python3-leapp should be
+# installed
+Conflicts:      python3-%{name}
```

Comment:
hmm, so an update on RHEL8 would conflict and you would have to remove it manually?

Scenario:
- I install leapp on RHEL8 to check upgrades and wait for a bug fix.
- Next release I update leapp on that RHEL8 machine and get a conflict
- Manually remove python3-leapp and install leapp again

That is what I am inferring from this `Conflicts` directive.


What about:
```
Conflicts: framework-version < 2.0
```

Reply:
The scenario you wrote is invalid. You install leapp on RHEL 8, so you can update packages without problems. This is only about scenario: you have upgraded from rhel 7 to rhel 8, so you have the old el7 packages still. Remove these manually and than install leapp for rhel8 (or update using the `--allowerasing` option. 

However, I believe we would like to be super strict and ensure even the `--allowerasing` will not work. Look at the gdoc I shared yesterday (see the jira) to see reasoning behind. It's going to be discussed with Pavel to see his opinion on user experience. I will do a mtg the next week once you all are familiar with the document. The conflicts are set between python2-leapp and python3-leapp packages - we do not want to provide python2-leapp (or anything for python2) in packages build for el8.

Reply:
duh - I was looking at the wrong thing. Of course this is correct.